src/components/layout/Nav.jsx

```diff
@@ -5,6 +5,7 @@ import SignUp from './SignUp';
 import styles from './Nav.module.css';
 import { useAuth } from '../../AuthContext';
 import logo from '../../assets/logos/blue.png';
+// import Profile from '../../pages/UserDashboard/Profile';
```

Comment:
a the console logs

Reply:
Where are the console logs, Laura? Do you mean to remove the commented code?